MainGame.py

```diff
@@ -24,3 +25,32 @@ def GetUserName(self, greeting: str) -> str:
             else:
                 print(
                     f"Your input '{userName}' is not valid. Please try again")
+
+    def Turn(self, player) -> None:
+        row: int = self.GetRowInput(player)
+
+    def GetRowInput(self, player) -> int:
+        while True:
+            try:
+                rowInputString: str = input(
+                    f"{player}: Select a row to put your coin:")
+                rowInput = int(rowInputString)
+
+                return rowInput, False
+            except ValueError:
+                print("Your Value is invalid. Try again!")
+
+    def Round(self) -> None:
+        player: Player = self.playerOne.name
```

Comment:
`player` is a `str` here :) I would also suggest to rename the variables appropriately.